playbooks/installation/index.adoc

```diff
@@ -408,7 +425,7 @@ Register your Ansible Control Host and install the appropriate repos
 ----
 subscription-manager register --username bob@myorg.com --password='mypassword'
 subscription-manager attach --pool=8a85f...
-subscription-manager repos --disable="*" --enable="rhel-7-server-rpms" --enable="rhel-7-server-extras-rpms" --enable="rhel-7-server-ose-3.6-rpms"
+subscription-manager repos --disable="*" --enable="rhel-7-server-rpms" --enable="rhel-7-server-extras-rpms" --enable="rhel-7-server-ose-3.7-rpms"
```

Comment:
Should also include `--enable="rhel-7-fast-datapath-rpms"`

Reply:
Good catch. Updated.